core/domain/classifier_domain.py

```diff
@@ -331,9 +331,10 @@ def validate(self):
 
         if self.status not in feconf.ALLOWED_TRAINING_JOB_STATUSES:
             raise utils.ValidationError(
-                'Expected status to be in %s, received %s' %
-                feconf.ALLOWED_TRAINING_JOB_STATUSES,
```

Comment:
Because the test suite fails when tests are written for this condition. It gives the error as "not enough arguments for format string". I myself was stuck here for a long time here and was not willing to change anything in the original script file. Can you please confirm this issue? I guess feconf.ALLOWED_TRAINING_JOB_STATUSES would return like 4 values. I even tried incorporating this but all in vain.

Reply:
The issue is because the two string formats `%s` are present and the values are not put in bracket. It can be fixed by replacing the lines to:

```
        if self.status not in feconf.ALLOWED_TRAINING_JOB_STATUSES:
            raise utils.ValidationError(
                'Expected status to be in %s, received %s' %
                (feconf.ALLOWED_TRAINING_JOB_STATUSES, self.exp_version))
```

Make this change as well change the tests accordingly (You will need to check for `Expected status to be in` instead of `Expected status value not passed, it should be one` in all the tests)

Reply:
Cool, it works. Thanks a lot